pkg/ai/ai_test.go

```diff
@@ -0,0 +1,328 @@
+package ai
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/gokultp/go-four-in-a-row/pkg/game"
+)
+
+func TestMakeMove(t *testing.T) {
+	g := game.NewGame(10, 10)
```

Comment:
It was better if you could use github.com/smartystreets/goconvey/convey for unit tests.

Reply:
Interesting, that wouldn't be too hard. What do you like about it? is it the test syntax or the browser GUI?